tasks.py

```diff
@@ -1138,7 +1138,7 @@ def service(
 
             if (
                 service_name == "mercury"
-                and os.environ.get("CI") is False  # noqa: TID251
+                and os.environ.get("CI") != "true"  # noqa: TID251
```

Comment:
If you use a pydantic `BaseSettings` module you can pull this environment variable and take advantage of pydantic bool parsing logic.

https://docs.pydantic.dev/1.10/usage/types/#booleans

```python
from typing import Optional
from pydantic.v1 import BaseSettings

... 

   class EnvVars(BaseSettings):
    ci: Optional[bool] = None

   service_names = set(names)
    if markers:
        for test_deps in _get_marker_dependencies(markers):
            service_names.update(test_deps.services)
    if service_names:
        print(f"  Starting services for {', '.join(service_names)} ...")
        for service_name in service_names:
            cmds = []

            if (
                service_name == "mercury"
                and EnvVars().ci is not True
```

Reply:
good idea, fixed.